File: Examples/DotNET/SDK/VisualBasic/Drawing/GetAllObjects.vb
Imports Aspose.Cloud

Namespace Drawing
    Class GetAllObjects
        Public Shared Sub Run()
            Dim subdirectory As String = "Drawing"

            ' The path to the documents directory.
            Dim dataDir As String = (Common.GetDataDir() & subdirectory) + "\"

            Dim inputfile As String = "testfile.doc"

            ' Upload input file from local directory to Cloud Storage
            Common.UploadFile(inputfile, subdirectory)

            ' Get All Drawing Objects
            Dim wordsDrawingObjectsResponse As WordsDrawingObjectsResponse = Common.WordsService.WordsDrawingObjects.ReadDocumentDrawingObjectsCommonInfo(inputfile, Common.FOLDER)

            Console.WriteLine((Convert.ToString(vbLf) & inputfile) + " have " + wordsDrawingObjectsResponse.DrawingObjects.List.Count + " drawing objects.")
        End Sub
    End Class
End Namespace

File: Examples/DotNET/SDK/CSharp/Drawing/GetAllObjects.cs
using System;
using Aspose.Cloud;

namespace CSharp.Drawing
{
    class GetAllObjects
    {
        public static void Run()
        {
            string subdirectory = "Drawing";

            // The path to the documents directory.
            string dataDir = Common.GetDataDir() + subdirectory + "\\";

            string inputfile = "testfile.doc";

            // Upload input file from local directory to Cloud Storage
            Common.UploadFile(inputfile, subdirectory);

            // Get All Drawing Objects
            WordsDrawingObjectsResponse wordsDrawingObjectsResponse = Common.WordsService.WordsDrawingObjects.ReadDocumentDrawingObjectsCommonInfo(inputfile, Common.FOLDER);

            Console.WriteLine("\n" + inputfile + " have " + wordsDrawingObjectsResponse.DrawingObjects.List.Count + " drawing objects.");
        }
    }
}
